From 327ac105466a7f692b3af0ed83961b59ccc91747 Mon Sep 17 00:00:00 2001
From: Ankit Agarwal <ankit.agarwal@intel.com>
Date: Thu, 20 Jun 2024 10:34:20 +0530
Subject: [PATCH] Updated compressing binary in A15.

In A15, its using gzip binary instead of minigzip.

Tests: Prepared EB, it is successfull.

Tracked-On: NA
Signed-off-by: Ankit Agarwal <ankit.agarwal@intel.com>
---
 releasetools/intel_common.py | 4 ++--
 tasks/liveimage.mk           | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/releasetools/intel_common.py b/releasetools/intel_common.py
index 32184f6..17a37a2 100644
--- a/releasetools/intel_common.py
+++ b/releasetools/intel_common.py
@@ -600,7 +600,7 @@ def GetFastbootImage(unpack_dir, info_dict=None):
         shutil.rmtree(ramdisk_tmp)
         raise exc
 
-    cmd2 = ["minigzip"]
+    cmd2 = ["gzip"]
     try:
         p2 = common.Run(
             cmd2, stdin=p1.stdout, stdout=ramdisk_img.file.fileno())
@@ -613,7 +613,7 @@ def GetFastbootImage(unpack_dir, info_dict=None):
     p2.communicate()
     p1.wait()
     assert p1.returncode == 0, "mkbootfs of fastboot ramdisk failed"
-    assert p2.returncode == 0, "minigzip of fastboot ramdisk failed"
+    assert p2.returncode == 0, "gzip of fastboot ramdisk failed"
 
     # use MKBOOTIMG from environ, or "mkbootimg" if empty or not set
     mkbootimg = os.getenv('MKBOOTIMG') or "mkbootimg"
diff --git a/tasks/liveimage.mk b/tasks/liveimage.mk
index 7edc301..0cbf7a2 100644
--- a/tasks/liveimage.mk
+++ b/tasks/liveimage.mk
@@ -50,7 +50,7 @@ $(live_ramdisk): \
 		$(INSTALLED_RAMDISK_TARGET) \
 		$(MKBOOTFS) \
 		$(live_initrc) \
-		| $(MINIGZIP) $(ACP) \
+		| $(GZIP) $(ACP) \
 
 	$(hide) mkdir -p $(dir $@)
 	$(hide) rm -rf $(ramdisk_dir)
@@ -63,7 +63,7 @@ $(live_ramdisk): \
 	$(hide) echo "import init.live.rc" >> $(ramdisk_dir)/init.rc
 	$(hide) sed -i -r 's/^[\t ]*(mount_all|mount yaffs|mount ext).*//g' $(ramdisk_dir)/init*.rc
 	$(hide) $(ACP) $(live_initrc) $(ramdisk_dir)
-	$(hide) $(MKBOOTFS) $(ramdisk_dir) | $(MINIGZIP) > $@
+	$(hide) $(MKBOOTFS) $(ramdisk_dir) | $(GZIP) > $@
 
 $(live_bootimage): \
 		$(INTEL_PATH_BUILD)/tasks/liveimage.mk \
-- 
2.34.1

